README.md

```diff
@@ -45,7 +45,7 @@ $ sudo vi /etc/hosts
 
 Now that you've generated the certificates and added the `example.com` host entries to `/etc/hosts`, you can start Play itself.
 
-This application is not run with `activator` -- you should run it with `./play` instead, as there are a number of system properties required to use it effectively.
+This application is not run with `sbt` -- you should run it with `./play` instead, as there are a number of system properties required to use it effectively.
```

Comment:
Out of curiosity, have you tried running it with `gradle`? I have not and I wonder if it'll work or fail.

Reply:
no, I haven't